halo2_proofs/src/plonk/circuit.rs

```diff
@@ -532,6 +610,14 @@ pub trait Assignment<F: Field> {
         NR: Into<String>,
         N: FnOnce() -> NR;
 
+    // /// Allows the developer to include an annotation for an specific column within a `Region`.
+    // ///
+    // /// This is usually useful for debugging circuit failures.
+    // fn annotate_column<A, AR>(&mut self, annotation: A, column: Column<Any>)
+    //     where
+    //         A: FnOnce() -> AR,
+    //         AR: Into<String>;
+
```

Comment:
TODO?

Reply:
https://github.com/zcash/halo2/pull/706 is ready now.  This should be closer to what we will have in the future.